coremltools/test/test_keras2_numeric.py

```diff
@@ -1877,23 +1877,23 @@ def test_tiny_mobilenet_arch(self):
         img_input = Input(shape=(32,32,3))
         x = Conv2D(4, (3,3), padding='same', use_bias=False, strides=(2,2), name='conv1')(img_input)
         x = BatchNormalization(axis=-1, name='conv1_bn')(x)
-        x = Activation('relu', name='conv1_relu')(x)
+        x = Activation(relu6, name='conv1_relu')(x)
```

Comment:
Seems like we'd want to test both relu and relu6. Do we have other unit tests that test relu?

Reply:
+1